templates/deb/postinst_upgrade.sh.erb

```diff
@@ -8,18 +8,26 @@ after_upgrade() {
 <%=  script(:after_upgrade) %>
 <% end -%>
 
-<% if attributes[:deb_systemd] -%>
+<%# if any systemd services specified, loop through and start them -%>
+<% unless attributes[:deb_systemd].empty? -%>
 systemctl --system daemon-reload >/dev/null || true
-if ! systemctl is-enabled <%= attributes[:deb_systemd] %> >/dev/null 
+sysctl=$(command -v deb-systemd-invoke || echo systemctl)
+<% attributes[:deb_systemd].each do |service| -%>
+if ! systemctl is-enabled <%= service %> >/dev/null 
 then
-    systemctl enable <%= attributes[:deb_systemd] %> >/dev/null || true
-    systemctl start <%= attributes[:deb_systemd] %> >/dev/null || true
+<% if attributes[:deb_systemd_enable?]-%>
+    systemctl enable <%= service %> >/dev/null || true
```

Comment:
did you mean `$sysctl` instead of `systemctl` here?

Reply:
same comment as above